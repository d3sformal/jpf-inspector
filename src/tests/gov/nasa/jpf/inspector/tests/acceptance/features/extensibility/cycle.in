# No cycles:
hi
hi2
# Cycles:
cycle
cycle2
cycle3
cycle4
# With "&&":
# No cycles:
hi && hi
# Two cycles:
cycle && cycle
# One ok, one cycle
hi2 && cycle3
cycle2 && hi
