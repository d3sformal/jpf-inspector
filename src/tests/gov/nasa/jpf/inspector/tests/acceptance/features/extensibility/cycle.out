Hello to you, too, dear user! The JPF Inspector command loop is working!
Hello to you, too, dear user! The JPF Inspector command loop is working!
ERR: The alias 'cycle2' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
ERR: The alias 'cycle2' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
ERR: The alias 'cycle3' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
ERR: The alias 'cycle4' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
Hello to you, too, dear user! The JPF Inspector command loop is working!
Hello to you, too, dear user! The JPF Inspector command loop is working!
ERR: The alias 'cycle2' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
ERR: The alias 'cycle2' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
Hello to you, too, dear user! The JPF Inspector command loop is working!
ERR: The alias 'cycle3' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
ERR: The alias 'cycle2' contains itself in its own value (perhaps after several expansions). This is a cyclic dependency which cannot be resolved. The alias cannot be used.
Hello to you, too, dear user! The JPF Inspector command loop is working!